docs/development-setup.md

```diff
@@ -24,23 +24,22 @@ This short guide focuses on setting up the project using IntelliJ instead of And
 
 1. Install docker and docker-compose
 2. `sudo docker-compose rm`
-2. `sudo docker-compose build`
-2. `sudo docker-compose up --force-recreate`
-3. Open Adminer: [http://localhost:5001](http://127.0.0.1:5001/?pgsql=db-postgis&username=postgres&db=ehrenamtskarte)
-
+3. `sudo docker-compose build`
+4. `sudo docker-compose up --force-recreate`
+5. Open Adminer: [http://localhost:5001](http://127.0.0.1:5001/?pgsql=db-postgis&username=postgres&db=ehrenamtskarte)
    The credentials are:
-
    |Property|Value|
    |---|---|
    |Host (within Docker)|db-postgis|
    |Username|postgres|
    |Password|postgres|
    |Database|ehrenamtskarte|
-4. Install JDK8
-5. Run the backend: `cd backend && ./gradlew run` or `.\backend\gradlew.bat run` on Windows
-6. Take a look at the martin endpoints: [http://localhost:5002/tiles/accepting_stores/index.json](http://localhost:5002/tiles/accepting_stores/index.json) and [http://localhost:5002/tiles/accepting_stores/rpc/index.json](http://localhost:5002/tiles/accepting_stores/rpc/index.json). The data shown on the map is fetched from a hardcoded url and is not using the data from the local martin!
-7. Take a look at the style by viewing the test map: [http://localhost:5002](http://localhost:5002)
-8. Take a look at the backend: [http://localhost:7000](http://localhost:7000) (The public version is available at api.ehrenamtskarte.app)
+6. Install JDK8
+7. Run the backend: `cd backend && ./gradlew run --args="execute"` or `.\backend\gradlew.bat run --args="execute"` on Windows
+8. Create an admin account using `./gradlew run --args="create-admin <email> <password>"`
```

Comment:
Does it make sense to create an idea run config for this as well with a dummy admin account?

Reply:
Upon executing `./gradlew run --args="create-admin asfd@asdf.de 1234"` I get:
> INSERT INTO categories ("id", "name")
VALUES (8, 'Essen/Trinken/Gastronomie') ON CONFLICT ("id")
    DO UPDATE SET
        "name"='Essen/Trinken/Gastronomie';
INSERT INTO categories ("id", "name")
VALUES (9, 'Sonstiges') ON CONFLICT ("id")
    DO UPDATE SET
        "name"='Sonstiges';
Exception in thread "main" java.lang.IllegalStateException: No transaction in context.
        at org.jetbrains.exposed.sql.transactions.TransactionManager$Companion.current(TransactionApi.kt:98)
        at org.jetbrains.exposed.dao.EntityClass.new(EntityClass.kt:237)
        at org.jetbrains.exposed.dao.EntityClass.new(EntityClass.kt:220)
        at app.ehrenamtskarte.backend.auth.database.repos.AdministratorsRepository.insert(AdministratorsRepository.kt:18)
        at app.ehrenamtskarte.backend.common.database.Database$Companion.createAccount(Database.kt:35)
        at app.ehrenamtskarte.backend.CreateAdmin.run(EntryPoint.kt:50)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:171)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:180)
        at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:16)
        at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:395)
        at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:392)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:410)
        at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:435)
        at app.ehrenamtskarte.backend.EntryPointKt.main(EntryPoint.kt:62)
FAILURE: Build failed with an exception.